fix: updated instructions for install-and-set-up-mongoose (#37908)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.english.md

## Description
<section id='description'>

> Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI ( must be saved as a STRING) in the private <code>.env</code> file as MONGO_URI. Connect to the database using the following syntax:

Reviewer: ```suggestion
Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI in the private <code>.env</code> file as MONGO_URI. Surround the the URI with single of double quotes and there should be no space between the variable and the `=`. Connect to the database using the following syntax:
```

Author: typo 'of'. 'single or double'.

> Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI in the private <code>.env</code> file as MONGO_URI. Surround the the URI with single or double quotes and there should be no space between the variable and the `=` and the value. Connect to the database using the following syntax:

Reviewer: Thanks for catching my typo on the other suggestion.  After reading this again, I think we need one more change.  You can commit this if you agree.

```suggestion
Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI in the private <code>.env</code> file as MONGO_URI. Surround the the URI with single or double quotes and make sure no space exists between both the variable and the `=` and the value and `=`. Connect to the database using the following syntax:
```
